### PR TITLE
Password check

### PR DESCRIPTION
**Pull request by jantob6637**
authors: @jantob6637 , @Rogal255

### CODE REVIEW

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @jantob6637

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @Rogal255

**Comment by ziobron** [comment]
✅ CI Passed - Task: password-check-bonus
🏅 Granted 4 XP for @jantob6637

File: homework/password-check/validation.cpp [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
-// TODO: Put implementations here
\ No newline at end of file
+
+std::string getErrorMessage(ErrorCode error)
+{
+    switch (error)
+    {
+    case ErrorCode::Ok:
+    return "Ok";
+
```
Comment (nauka-programowania-MA) [inline homework/password-check/validation.cpp:9]: Redundant new line between each case.

**Review by nauka-programowania-MA — APPROVED** [review]
BEAUTIFUL!!!! I really enjoy your code style.

**Pull request closed without merging**